### DISCUSSION TITLE
Transmission Haugene 3.00 black theme

### DISCUSSION THREAD

**Discussion by ghost** [haugene/docker-transmission-openvpn#2417] [Ideas]
Hi !

I coded the Transmission 3.00 common.css file to display the Transmission interface in black theme.

common.css file is to put on Transmission Haugene Docker Image /usr/share/transmission/web/
[common.zip](https://github.com/haugene/docker-transmission-openvpn/files/10032219/common.zip)
![capture](https://user-images.githubusercontent.com/90903325/202539465-1e44e39a-aa24-42b4-ac62-ff92b67b70c7.jpg)

**Comment by pkishino** [comment]
nice work,
I recommend using the Flood UI personally
